### DISCUSSION TITLE
Atari Lynx Support

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#951] [Ideas & Suggestions]
This one just always seemed to be a big omission to me, given that we support FAR more obscure, and FAR more popular stuff, it seems a bit lost in the middle.
There are other solutions at least, like the JoeyLynx, and a Pi hat that can be used for dumping. But there's plenty of other solutions for many here, and getting it all one place would be really nice. I think it's about the only one 'most people have heard of' that we're missing haha.

**Comment by partlyhuman** [comment]
Disclaimer: I've only been looking at this for a few minutes, and I could be way off base.
It seems like the pitch of the card edge is 1.59mm which seems a bit odd, it might be difficult to source... digikey shows only 1.57mm pitch and they're all non-stocked. However, the folks selling a programmer/dumper HAT for Raspberry Pi also sell a card edge connector: https://k-retro.com/atari-lynx-cart-dumpers-programmers/202-cart-connector-and-adapter-kit-for-atari-lynx.html

I think with this connector in hand, it'd be a fun project! I used to have a Lynx when I was a kid but god knows what happened to it.

**Reply by partlyhuman** [reply]
Some discussion about finding an appropriate card edge connector by the devs: https://forums.atariage.com/topic/289004-atari-lynx-programmer-pi-hat-orders/page/10/#comment-4316179

Seems like they might have found a supplier but aren't going to share, that's all fine since it's available from their shop at a reasonable price. https://forums.atariage.com/topic/289004-atari-lynx-programmer-pi-hat-orders/page/6/#comment-4270696

**Reply by partlyhuman** [reply]
Connectors are on the way, hoping to play with this idea very soon! I also ordered a Lynx flashcart so hopefully we can use OSCR as a flasher too.
